units/systemd-udevd.service.in

```diff
@@ -21,7 +21,10 @@ Sockets=systemd-udevd-control.socket systemd-udevd-kernel.socket
 Restart=always
 RestartSec=0
 ExecStart=@rootlibexecdir@/systemd-udevd
-MountFlags=slave
 KillMode=mixed
 WatchdogSec=3min
 TasksMax=infinity
+MountFlags=slave
+MemoryDenyWriteExecute=yes
+RestrictRealtime=yes
+RestrictAddressFamilies=AF_UNIX AF_NETLINK
```

Comment:
> Note that this change turns off AF_INET/AF_INET6 from udevd, thus effectively
turning of networking from udev rule commands.

Shouldn't we update `udev(7)`?

(I believe some people use `wget <some-internal-host-or-local-host>/get-ugly-json` or so. Why should we break this?)